scripts/index.js

```diff
@@ -1 +1,158 @@
-// add your solutions here
\ No newline at end of file
+// add your solutions here
+function positiveNumbers(ar) {
+    const newArray = []
+    ar.forEach(function(num){
+        if (num >= 0){
+            newArray.push(num);
+        }
+    })
+    return newArray;
+}
+console.log(positiveNumbers([1, 2, 4, -2, -1, 0, 5, -9, -6, -9.8, 8.7, 3]));
+
+//is positive with filter
+function positiveFilter(num){
+    if (num >= 0){
```

Comment:
Nice way to put the idea of "positive" into a function. Would it be possible to reduce this down to a single `return` statement? (That is, skip the `if`)